### PR TITLE
JCLOUDS-1460: Add support for aws ec2 t3 instance types

### PR DESCRIPTION
**Pull request by pimuzzo**
This is the [jira](https://issues.apache.org/jira/browse/JCLOUDS-1460).
Thank you for the review!

### CODE REVIEW

File: apis/ec2/src/main/java/org/jclouds/ec2/compute/domain/EC2HardwareBuilder.java [hunk 1]
```diff
@@ -429,6 +435,69 @@ public static EC2HardwareBuilder t2_large() {
             .processors(ImmutableList.of(new Processor(1.0, 0.4))).rootDeviceType(RootDeviceType.EBS);
    }
 
+   /**
+    * @see InstanceType#T3_NANO
+    */
+   public static EC2HardwareBuilder t3_nano() {
+      return new EC2HardwareBuilder(InstanceType.T3_NANO).t1()
+              .ram(512)
+              .processors(ImmutableList.of(new Processor(2.0, 1.0)));
```
Comment (pimuzzo) [inline apis/ec2/src/main/java/org/jclouds/ec2/compute/domain/EC2HardwareBuilder.java]: Where can I found the correct speed?

Comment (trentontrees) [reply apis/ec2/src/main/java/org/jclouds/ec2/compute/domain/EC2HardwareBuilder.java]: AWS Instance Type reference page:
https://aws.amazon.com/ec2/instance-types/

**Comment by andreaturli** [comment]
I'm confused guys, does #1245 supersede this one?

**Comment by trentontrees** [comment]
We did our work separately.  The code in PR #1245 is slightly different than this.  I do think that #1245 supersedes this, but that PR is current blocked because of a build error.  The `t3()` method in this PR is slightly different as well.

**Comment by pimuzzo** [comment]
I think too that the @trentontrees 's PR supersedes this one :peace_symbol: 
@andreaturli you can close this one, there will be other opportunities to collaborate

**Comment by andreaturli** [comment]
thanks @pimuzzo for the clarification

Feel free to close this yourself, and maybe help with the review of #1245, as you are waiting for (some) same features contained there!

Thanks!

**Pull request closed without merging**